docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -259,12 +261,27 @@ Custom Metrics
 ==============
 You can also pass your own custom metric to relevant checks and suites.
 
-For computer vision the custom metrics should support the
-`Ignite Metric <https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric>`__ API.
+For tabular metrics and vision classification tasks the custom metrics function should follow the
+`sklearn scorer <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html>`__ API, which is
+a function that accepts the parameters: (model, x, y_true), and returns a score with the convention that higher is
+better.
+
+For other computer vision tasks, the custom metrics should be a custom metric class. Deepchecks Custom Metric classes
+are inspired by `Ignite metrics <https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric>`__ and implement
+the following methods: ``reset``, ``update`` and ``compute``.
+
+    * ``reset`` - resets the metric to its initial state, creates or resets any internal variables.
+    * ``update`` - updates the metric's internal state based on the provided. The method's signature
+        should be ``update(self, y_pred, y_true)``, where ``y_pred`` is the model's output and ``y_true`` is the ground
+        truth, both given  as lists of numpy objets. For example, for object detection the label would be a list
+        where each element is a numpy array of bounding boxes annotations, and the prediction would be a list where each
+        element is a numpy array of bounding boxes predictions, both in the
+        :doc:`deepchecks format </user-guide/vision/supported_tasks_and_formats.rst>`.
+    * ``compute`` - returns the metric's value based on the internal state.
```

Comment:
should return either a single number of a list of numbers representing value per class.
maybe also mention they must be numpy or int (not tensors)